core/src/main/java/org/elasticsearch/index/query/GeoDistanceRangeQueryBuilder.java

```diff
@@ -92,6 +98,13 @@ public GeoPoint point() {
     }
 
     public GeoDistanceRangeQueryBuilder from(Object from) {
```

Comment:
The only reason I can think of is that also lt(), lte(), gt() and gte() would also need two versions, quiet a lot of setters then. We can also remove those, rely on the API user to explicitely call includeUpper/Lower then, removes convenience though.